app/index.js

```diff
@@ -253,7 +261,10 @@ module.exports = yeoman.Base.extend({
       this.log();
     } else {
       this.log(g.f('  Create a model in your app'));
-      this.log(chalk.green('    $ ' + cmd + ' loopback:model'));
+      if (cmd === 'loopback-cli')
```

Comment:
I think you're right! Too bad. ;( It's love/hate since we get some consistency. I don't like it when it doesn't let me `cb && cb()` either, but rules are rules. ;p